backend/wait-for-db.py

```diff
@@ -0,0 +1,31 @@
+import os
+import asyncio
+import asyncpg
+import time
+
+database_url = os.environ.get("DATABASE_URL")
+database_url = "postgresql://user:password@db:5432/llama_app_db"
```

Comment:
It would be better to pull the database url from `settings.DATABASE_URL` (where settings is from `from app.core.config import settings`).

The value in `settings.DATABASE_URL` has some pre-processing done to it *(as seen [here](https://github.com/run-llama/sec-insights/blob/main/backend/app/core/config.py#L110-L118))* so it uses the `asyncpg` postgres driver.

Reply:
changing the value of database_url to settings.DATABASE_URL is causing errors. Seems like an issue on my end will check and get back at the earliest.

Reply:
made necessary changes